pkg/api/api.go

```diff
@@ -32,4 +33,12 @@ const (
 	TrustedCABundleMountFile            = "tls-ca-bundle.pem"
 	OCCLIDownloadsCustomResourceName    = "oc-cli-downloads"
 	ODOCLIDownloadsCustomResourceName   = "odo-cli-downloads"
+
+	ConsoleContainerPortName    = "https"
+	ConsoleContainerPort        = 443
+	ConsoleContainerTargetPort  = 8443
+	RedirectContainerPortName   = "redirect"
```

Comment:
Shall we call it `custom-route-redirect`, for clarity?

Reply:
Wanted to keep the name simple here